vendor/bat-native-ledger/src/bat/ledger/internal/state/state.cc

```diff
@@ -240,16 +272,23 @@ std::vector<uint8_t> State::GetRecoverySeed() {
 }
 
 void State::SetRecoverySeed(const std::vector<uint8_t>& seed) {
+  const std::string seed_string = base::Base64Encode(seed);
+  std::string event_string;
+  if (seed.size() > 1) {
```

Comment:
Should this be a `DCHECK` or is a seed size <= 1 also valid?

Reply:
in theory seed can get corrupted, so we need this check